source/renderer/app/components/staking/stake-pools/helpers.js

```diff
@@ -13,9 +14,9 @@ const stakePoolsListSearch = (stakePool: StakePool, rawSearch: string) => {
 };
 
 export const getFilteredStakePoolsList = (
-  stakePoolsList: StakePoolsListType,
+  stakePoolsList: Array<StakePool>,
   search: string
-): StakePoolsListType =>
-  stakePoolsList.filter((stakePool: StakePool) =>
+): Array<StakePool> =>
+  stakePoolsList.filter((stakePool: Object) =>
```

Comment:
@daniloprates  Why is here type `Object`?

Reply:
✅